integrations/slack/src/index.ts

```diff
@@ -26,8 +27,115 @@ const handleSpaceContentUpdated: EventCallback<
         return;
     }
 
+    const { data: semanticChanges } = await api.spaces.getRevisionSemanticChanges(
+        event.spaceId,
+        event.revisionId,
+        {
+            // Ignore git metadata and custom field changes
+            metadata: false,
+        }
+    );
+
+    if (semanticChanges.changes.length === 0) {
+        // No changes to notify about
+        return;
+    }
+
     const { data: space } = await api.spaces.getSpaceById(event.spaceId);
 
+    /*
+     * Build a notification that looks something like this:
+     *
+     *    Content of *Space* has been updated.
+     *
+     *    Summary of Changes:
+     *    • New pages: Page1, Page2
+     *    • Modified pages: Page3
+     *    • Deleted pages: Page4, Page5
+     *    • Moved pages: Page6
+     *    • New files: File1, File2
+     *    • Modified files: File3
+     *    • Deleted files: File4
+     *
+     *    And another X changes are not listed here.
+     */
+
+    const createdPages = [];
+    const editedPages = [];
+    const deletedPages = [];
+    const movedPages = [];
+    const createdFiles = [];
+    const editedFiles = [];
+    const deletedFiles = [];
+
+    semanticChanges.changes.forEach((change) => {
+        switch (change.type) {
+            case RevisionSemanticChangeType.PageCreated:
+                createdPages.push(change.page.title);
+                break;
+            case RevisionSemanticChangeType.PageEdited:
+                editedPages.push(change.page.title);
+                break;
+            case RevisionSemanticChangeType.PageDeleted:
+                deletedPages.push(change.page.title);
+                break;
+            case RevisionSemanticChangeType.PageMoved:
+                movedPages.push(change.page.title);
+                break;
+            case RevisionSemanticChangeType.FileCreated:
+                createdFiles.push(change.file.name);
+                break;
+            case RevisionSemanticChangeType.FileEdited:
+                editedFiles.push(change.file.name);
+                break;
+            case RevisionSemanticChangeType.FileDeleted:
+                deletedFiles.push(change.file.name);
+                break;
+            default:
+                break;
+        }
+    });
+
+    let notificationText = `Content of *${space.title || 'Space'}* has been updated.`;
```

Comment:
Samy suggested to remove links as we don't know whether the recipient of the notification has access to the space or not. Up to discussion. Easy to reintroduce if we prefer that.